src/client/auth/oidc.rs

```diff
@@ -69,33 +143,77 @@ pub struct IdpServerInfo {
     pub request_scopes: Option<Vec<String>>,
 }
 
+#[derive(Debug)]
```

Comment:
This should probably be `#[non_exhaustive]` so we can add fields later if needed.